vizro-core/docs/pages/tutorials/first-dashboard.md

```diff
@@ -1,51 +1,56 @@
-# Get started with Vizro
+# A first dashboard
 
-This tutorial introduces Vizro. It is a step-by-step guide to create a first dashboard and is designed to equip you with the knowledge to explore the documentation further.
+This is a short tutorial for you to create your first dashboard, showing you the basic setup so you can explore Vizro further.
 
-If you want a more complete tutorial exploring more of the available features, please see the [Explore Vizro tutorial](../tutorials/explore-components.md).
+Once you've completed this tutorial, the following [Explore Vizro tutorial](../tutorials/explore-components.md) creates a more complex dashboard so you can explore Vizro's features.
 
 
-## Let's get started!
+## Get started
 
 ### 1. Install Vizro and its dependencies
 
-If you haven't already installed Vizro, follow the [installation guide](../user-guides/install.md) to do so inside a virtual environment.
+If you haven't already installed Vizro, follow the [installation guide](../user-guides/install.md).
 
-If you consider yourself a beginner to python and/or virtual environments, there is also a section that avoids any use of terminals and relies only upon a graphical user interface.
+### 2. Open a Jupyter Notebook
```

Comment:
I wonder whether Jupyter notebook is actually the right thing to recommend for a new user - maybe we should go for Jupyter lab instead, which is much nicer in pretty much every way, and I think is probably well-established enough now that it should be everyone's default choice.

No need to make the change now, but let's bear in mind for the future.